helm/operator/templates/console-clusterrolebinding.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: console-sa-binding
+  labels: {{- include "minio-operator.console-labels" . | nindent 4 }}
```

Comment:
Not sure if we want to include all labels here. This would also add the helm and selector labels to a clusterrole  The helm label is okay, but not sure if we would like the selector labels to be added to this object.

Reply:
That makes sense. I updated it to only apply the selector labels to the deployments and not to the other resources.